contracts/test/integration/IntegrationTestVoltV2.t.sol

```diff
@@ -0,0 +1,375 @@
+//SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+import {DSTest} from "../unit/utils/DSTest.sol";
+import {Vm} from "../unit/utils/Vm.sol";
+import {VoltV2} from "../../volt/VoltV2.sol";
+import {Core} from "../../core/Core.sol";
+import {MainnetAddresses} from "./fixtures/MainnetAddresses.sol";
+import {stdError} from "../unit/utils/StdLib.sol";
+
+contract IntegrationTestVoltV2 is DSTest {
```

Comment:
this should be a unit test imo, not an integration test

Reply:
I agree this could be a unit test for a lot of the logic, however we do have logic such as minting only being allowed with the `minter` role and setting a new `core` address only being allowed with a `governor` role, & I feel better about having these tested against mainnet state vs mimicking it in a unit test, although alternatively we could separate out these functions as an integration test, and have the others as a unit test